lib/index.js

```diff
@@ -53,19 +53,48 @@ function createMetricsApi({ metrics, log }) {
       name: 'kafka_cache_stream_error',
       help: 'A kafka error was encountered, likely causing connect failure'
     });
+    const consumerLagGauge = new metrics.Gauge({
+      name: 'kafka_cache_consumer_lag',
+      help: 'The consumer_lag statistics metric as reported by librdkafka (should only momentarily be > 0)',
+      labelNames: ['topic', 'partition']
+    });
 
     log.info('Prometheus metrics enabled!');
 
     return {
       incMissingKey: () => missingKeyCounter.inc(),
-      incStreamError: () => streamErrorCounter.inc()
+      incStreamError: () => streamErrorCounter.inc(),
+      onKafkaStats: (payload) => {
+        const stats = JSON.parse(payload.message.toString());
+
+        Object.keys(stats.topics).forEach(topicName => {
+          const partitions = stats.topics[topicName].partitions;
+
+          Object.keys(partitions).forEach(partitionNumber => {
+            // I'm not quite sure what this topic is reported for.
+            // I'm guessing it's got something to do with the bootstrap broker
+            // that also gets reported maybe?
+            if (partitionNumber === '-1') return;
+
+            const { consumer_lag } = partitions[partitionNumber];
+
+            consumerLagGauge.set({
+              topic: topicName,
+              partition: partitionNumber
+            }, consumer_lag);
+          });
+        });
+
+        log.debug({ stats }, 'onKafkaStats');
```

Comment:
What should we do with the rest of the stats that gets reported? Do we think we'll need them for debugging consumer lag situations?